vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageBlockCodeItem.kt

```diff
@@ -17,25 +17,28 @@
 package im.vector.app.features.home.room.detail.timeline.item
 
 import android.widget.TextView
+import androidx.appcompat.widget.AppCompatTextView
 import com.airbnb.epoxy.EpoxyAttribute
 import com.airbnb.epoxy.EpoxyModelClass
 import im.vector.app.R
 import im.vector.app.core.epoxy.onClick
 import im.vector.app.core.extensions.setTextOrHide
+import im.vector.app.core.extensions.setVectorText
+import im.vector.app.features.html.VectorCharSequence
 import me.saket.bettermovementmethod.BetterLinkMovementMethod
 
 @EpoxyModelClass(layout = R.layout.item_timeline_event_base)
 abstract class MessageBlockCodeItem : AbsMessageItem<MessageBlockCodeItem.Holder>() {
 
     @EpoxyAttribute
-    var message: CharSequence? = null
+    var message: VectorCharSequence? = null
 
     @EpoxyAttribute
     var editedSpan: CharSequence? = null
 
     override fun bind(holder: Holder) {
         super.bind(holder)
-        holder.messageView.text = message
+        holder.messageView.setVectorText(message)
```

Comment:
this is the fix for the crash in the code blocks (applying the same emoji fix we're using for the other message types)